mmda/parsers/symbol_scraper_parser.py

```diff
@@ -119,8 +118,8 @@ def _parse_row_head_tag(self, row_tag: str) -> Dict:
         match = re.match(pattern=r'<Line id=\"([0-9]+)\" BBOX=\"(.+)\">', string=row_tag)
         return {'id': int(match.group(1)), 'bbox': match.group(2)}
 
-    def _parse_word_head_tag(self, word_tag: str) -> Dict:
-        match = re.match(pattern=r'<Word id=\"([0-9]+)\">', string=word_tag)
+    def _parse_token_head_tag(self, token_tag: str) -> Dict:
```

Comment:
Just to confirm: as we are using "`token`" here, this means sscraper generated `word` might not be actual words -- probably they are split by lines or something else, right?

Reply:
that's right.  sscraper calls it `words` but we've decided to call these `tokens`